python/syntax.lark

```diff
@@ -36,13 +36,17 @@ arguments: "[" (name_type (" " name_type)*)? "]"
 anonymous_func: "(" arguments "->" NAME ":" (instruction (";" instruction)*) ")"
 char: "\\" ( escape_code | "{" /./ "}")
 escape_code: /[tnr]/
+tupledef: "(" NAME ("," NAME)* ")"
+tupleval: "(" value ("," value)* ")"
 
 // Boolean and number expressions, with order of operations.
 // Question mark "inlines" the branch, so we don't get nested
 // boolean_expression(or_expression(and_expression etc)).
 // Square brackets mean that the stuff inside it can appear 0 or 1 time. Same as
 // (whatever)?
 
+?types: NAME
+      | tupledef
```

Comment:
Maybe we could also add support for `->` for functions?

Reply:
? what, this is for types, before it was just `NAME` for I added support for tuples

Reply:
so that a function can accept a function as an argument, like `[convertInt:int -> int] ->`